docs/writing_a_parser_plugin/parser_plugin_overview.md

```diff
@@ -0,0 +1,75 @@
+# How to write a parser
+
+NOMAD uses parsers to convert raw data (for example, output from computational software, instruments,
+or electronic lab notebooks) into NOMAD's common Archive format. The following pages describe
+how to develop such a parser and integrate it within the NOMAD software. The goal is equip users
+with the required knowledge to contribute to and extend NOMAD.
+
+
+## Getting started
+
+In principle, it is possible to [develop a "local parser"](references/quick_parser_setup.md) that uses the *nomad-lab* package to parse raw data,
+without changing the NOMAD software itself. This allows a quick start for focusing on the parsing
+of the data itself, but is not relevant for full integration of your new parser into NOMAD.
+Here we are focused on developing parsers that will be integrated into the NOMAD software.
+For this, you will have to [install a development version of NOMAD](references/quick_installing_nomad_dev.md).
+
+
+### Parser organization
+
+The NOMAD parsers can be found within your local NOMAD git repo under
+`dependencies/parsers/`. The parsers are organized into the following individual projects
+(`dependencies/parsers/<parserproject>`) with their own corresponding repositories:
+
+* [atomistic](https://github.com/nomad-coe/atomistic-parsers) - Parsers for output from classical molecular simulations, e.g., from Gromacs, Lammps, etc.
+* [database](https://github.com/nomad-coe/database-parsers) - Parsers for various databases, e.g., OpenKim.
+* [eelsdb](https://github.com/nomad-coe/nomad-parser-eelsdb) - Parser for the EELS database (https://eelsdb.eu/; to be integrated in the database project).
+* [electronic](https://github.com/nomad-coe/electronic-parsers) - Parsers for output from electronic structure calculations, e.g., from Vasp, Fhiaims, etc.
```

Comment:
ab initio?

Reply:
@ndaelman-hu are you asking me or telling me? I cannot say myself. I was under the impression that some people (although I can't remember who) didn't like the term ab initio for some reason (perhaps it didn't fit some applications/calculations I don't know). Isn't electronic structure a broader/safer term? Or is it misleading for you?

Reply:
> Isn't electronic structure a broader/safer term?

It is, indeed. There are non-ab initio codes in electronic-parsers.